tests/cases/care_partner_platform/case_care_platform_participants.yaml

```diff
@@ -3985,261 +3994,53 @@ TestCarePartnerParticipantReviewEditOrAddVendorInfo:
-      Value: $AND_CLI_WEBSITE$
-  # Click on "save and close" button
-    - Type: click
-      Strategy: xpath
-      Id: $PAGE.care_platform_edit_participants_vendor.save_close_modal_button$
-  # Validate toast message and wait until toast is not visible
-      Vars:
-        TOAST_MESSAGE_SHOWN: Vendor created
     - Type: case
-      Value: CarePartnerToastValidation
-  # Steps to edit a vendor
-    - Type: click
-      Strategy: xpath
-      Id: $PAGE.care_platform_edit_participants_vendor.edit_created_vendor_button$
-  # Validate if "Edit vendor" is opened
-      Vars:
-        MODAL_NAME: Edit Vendor
-    - Type: get_text
-      Strategy: xpath
-      Id : $PAGE.care_platform_edit_participants_vendor.modal_title_label$
-      Greps:
-        - var: MODAL_TITLE
-          match: "(.*)"
-    - Type: assert
-      Role: desktopChrome
-      Asserts:
-        - Type: eq
-          Var: MODAL_TITLE
-          Value: $AND_CLI_MODAL_NAME$
-  # Remove current vendor name
-    - Type: clear_field_by_backspace
-      Strategy: id 
-      Id: $PAGE.care_platform_edit_participants_vendor.field_name_modal_input$
-  # Create a new vendor name
-    - Type: generate_unique_name
-      Name: 'New airport "L" Us'
-      ResultVar: NEW_NAME
-  # Fill input name vendor
-    - Type: send_keys
-      Strategy: id
-      Id: $PAGE.care_platform_edit_participants_vendor.field_name_modal_input$
-      Value: $AND_CLI_NEW_NAME$
-  # Closing "Edit Vendor" modal
-    - Type: click
-      Strategy: xpath
-      Id: $PAGE.care_platform_edit_participants_vendor.cancel_vendor_modal_button$
-  # Validate if Vendor is edited
-      Vars:
-        NEW_NAME: $AND_CLI_NAME$
-    - Type: get_text
-      Strategy: xpath
-      Id: $PAGE.care_platform_edit_participants_vendor.name_vendor_record$
-      Greps:
-        - var: CATEGORY_VENDOR_NAME
-          match: "(.*)"
-    - Type: assert
-      Role: desktopChrome
-      Asserts:
-        - Type: eq
-          Var: CATEGORY_VENDOR_NAME
-          Value: $AND_CLI_NAME$
-  # Opend "Edit Vendor" modal
-    - Type: click
-      Strategy: xpath
-      Id: $PAGE.care_platform_edit_participants_vendor.edit_created_vendor_button$
-  # Delete current vendor name
-    - Type: clear_field_by_backspace
-      Strategy: id 
-      Id: $PAGE.care_platform_edit_participants_vendor.field_name_modal_input$
-  # Create a new vendor name
-    - Type: generate_unique_name
-      Name: 'New airport "L" Us'
-      ResultVar: NEW_NAME
-  # Fill input name vendor
-    - Type: send_keys
-      Strategy: id
-      Id: $PAGE.care_platform_edit_participants_vendor.field_name_modal_input$
-      Value: $AND_CLI_NEW_NAME$
-  # Click "Save and close" button
-    - Type: click
-      Strategy: xpath
-      Id: $PAGE.care_platform_edit_participants_vendor.save_close_modal_button$
-  # Validate toast message
-      Vars:
-        TOAST_MESSAGE_SHOWN: Vendor saved
+      Value: AddEditOrDeleteParticipantVendorInfo
```

Comment:
Why did you make all these into a case helper? is this going to be reused?

Reply:
yes, it's going to be reused for the current test and the previously one, I asked about to keeping as test instead but @csanchez-nts told me that I have to handle like a helpertext.